Demo/Common/IntegrationExamples/Components/InstantPaymentComponentAdvancedFlowExample.swift

```diff
@@ -167,14 +167,14 @@ extension InstantPaymentComponentAdvancedFlowExample: ActionComponentDelegate {
     }
 }
 
-extension InstantPaymentComponentAdvancedFlowExample: PresentationDelegate {
+extension InstantPaymentComponentAdvancedFlow: PresentationDelegate {
     internal func present(component: PresentableComponent) {
         let componentViewController = viewController(for: component)
         presenter?.present(viewController: componentViewController, completion: nil)
     }
 }
 
-private extension InstantPaymentComponentAdvancedFlowExample {
+private extension InstantPaymentComponentAdvancedFlow {
```

Comment:
I know it's long but we have `example` keyword on the rest, why change only this one? Length warning?

Reply:
Yeah only because of the length warning 🙃 Any other ideas to keep the name shorter? 
I also thought to move them all under a `AdvanceFlowExample` enum "Namespace"

So we'd have:
```
enum AdvancedFlowExamples {}

extension AdvancedFlowExamples {
  class InstantPaymentComponent {
    ...
  }
}
```

What do you think? But maybe something for a different PR 🤔

Reply:
Will it not complain when it's like this :) ? End result will still be `AdvancedFlowExamples. InstantPaymentComponent` 
Otherwise could be nice indeed.

Reply:
The warning was about the class name length